### PR TITLE
Changed web api GetBigThumbnail to image processor

### PR DESCRIPTION
**Pull request by davidwincent**
in mediahelper.sevice.js and fileupload.controller.js

### CODE REVIEW

**Comment by nul800sebastiaan** [comment]
Thanks David, could you please link this to an issue at http://issues.umbraco.org so we can track it?

**Comment by davidwincent** [comment]
Shure!
http://issues.umbraco.org/issue/U4-7312

**Comment by davidwincent** [comment]
Made some update as discussed here: http://issues.umbraco.org/issue/U4-7312

File: src/Umbraco.Web.UI.Client/src/views/propertyeditors/fileupload/fileupload.controller.js [hunk 1]
```diff
@@ -144,31 +144,28 @@ function fileUploadController($scope, $element, $compile, imageHelper, fileManag
...
-                    return property.value;
+            assetsService.load(["lib/moment/moment-with-locales.js"]).then(
+                function () {
+                    //NOTE: The 'entity' can be either a normal media entity or an "entity" returned from the entityResource
+                    // they contain different data structures so if we need to query against it we need to be aware of this.
+                    mediaHelper.registerFileResolver("Umbraco.UploadField", function(property, entity, thumbnail){
+                        if (thumbnail) {
+                            if (mediaHelper.detectIfImageByExtension(property.value)) {
+                                //get default big thumbnail from image processor
+                                var thumbnailUrl = property.value + "?rnd=" + moment(entity.updateDate).format("YYYYMMDDHHmmss") + "&width=500";
```
Comment (Shazwazza) [inline src/Umbraco.Web.UI.Client/src/views/propertyeditors/fileupload/fileupload.controller.js]: So the purpose of the YYYYMMDDHHmmss is to invalidate client cache here right ? Why not make this just YYYYMMDDHHmm so we can at least have client cache for a minute ?

**Comment by Shazwazza** [comment]
Something in this PR breaks the normal upload control... I'm debugging this now.

**Comment by Shazwazza** [comment]
I've reverted this PR. We are releasing 7.3.2 very soon and unfortunately this won't be able to make it in there due to issues.

It would be much apprecated to submit another PR for the next version but please be sure to test:

* The standard image upload property editor
* The grid
* the image cropper
* any other place where a media image is being rendered.

File: src/Umbraco.Web.UI.Client/src/views/propertyeditors/fileupload/fileupload.controller.js (hunk 1 above)
Comment (davidwincent) [inline src/Umbraco.Web.UI.Client/src/views/propertyeditors/fileupload/fileupload.controller.js]: @Shazwazza - The rnd parameter should be the same as long as entity.updateDate does not change.

The idea is to have a valid cache until someone updates the entity.

I am making a new pull request based on dev-v7

**Pull request merged**